doc/devdocs/readme.md

```diff
@@ -75,12 +75,21 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 --add Microsoft.VisualStudio.Component.VC.ATL.Spectre
 ```
 
-### Compiling Source Code
+### Compiling Source Code from Visual Studio
```

Comment:
That sounds like compiling the code of Visual Studio

Reply:
I am not sure about that

"Compiling Source Code from Visual Studio" sounds right